src/main/java/tech/sourced/siva/SivaReader.java

```diff
@@ -0,0 +1,48 @@
+package tech.sourced.siva;
+
+import java.io.*;
+import java.nio.MappedByteBuffer;
+import java.nio.channels.FileChannel;
+
+public class SivaReader {
+
+    private final RandomAccessFile sivaFile;
+    private final FileChannel channel;
+
+    public SivaReader(File sivaFile) throws FileNotFoundException {
```

Comment:
Do you think it's consistent in API, to throw 2 different checked exception in reader?

For the client, what is the benefit of mandator extra clause of exception handing for none-recoverable conditions? Or am I missing something and this is actually a recoverable one?

As I client, it's quite convenient to have a single exception clause - if reading some .siva files failed, it will be just skipped.